packages/salesforcedx-vscode-replay-debugger/src/messages/i18n.ts

```diff
@@ -25,11 +25,14 @@ export const messages = {
     'The log file either is missing or does not have any log lines in it.',
   incorrect_log_levels_text:
     'The log must be generated with log categories Apex code at the FINEST level and Visualforce at the FINER level.',
-  up_to_five_checkpoints: 'You can set up to five checkpoints.',
+  up_to_five_checkpoints:
+    'You have %s of the maximum 5 enabled checkpoints. Remove or disable checkpoints until 5 or fewer are active.',
   checkpoints_can_only_be_on_valid_apex_source:
-    'Checkpoints can be set only on a valid line of Apex source.',
+    'Checkpoints may be set on a valid line of Apex source. URI=%s, Line=%s',
```

Comment:
Hi @JimSuplizio . We avoid "may" because it indicates permission. What is the reason for this change? Also, maybe we should give some context for why we're providing the values at the end of the message?

Reply:
I'll change it.